news/get-authors.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
```

Comment:
I recommend adding a docblock that explains the purpose of this script and any args it accepts (if any).